portfolio/src/main/webapp/statistics.html

```diff
@@ -172,13 +173,18 @@ <h4>Leave a marker on the map for protests you will be attending!</h4>
         </form>
       </div>
       <div id="clear"></div>
+      <br />
+      <br />
```

Comment:
Prefer to use CSS to create space (unless this was actually intended to create a line break, then nvm). It's more flexible and it'll automatically apply to all elements with the same class.

Reply:
This was used for a line break to create space between the map/chart and the end of the section.